annotators/emotion_detection/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM nvidia/cuda:12.1.1-base-ubuntu20.04
+
+RUN apt update
+RUN apt install -y python3.9
+RUN apt install -y git python3-pip
+
+WORKDIR /src
+
+COPY . /src
+RUN pip install -r requirements.txt
+
+RUN apt install -y ffmpeg libsm6 libxext6
+
+RUN pip install gdown
+
+RUN git clone https://github.com/anna-a-m/MultimodalERC repo
```

Comment:
pls pick up some particular commit because now it is not fixed at all

Reply:
particular commit picked